apps/parsley/src/components/DetailsMenu/DetailsMenuCard/Toggles/PrettyPrintToggle/index.tsx

```diff
@@ -1,26 +1,22 @@
 import { usePreferencesAnalytics } from "analytics";
-import { LogRenderingTypes } from "constants/enums";
 import { useLogContext } from "context/LogContext";
 import BaseToggle from "../BaseToggle";
 
 const PrettyPrintToggle: React.FC = () => {
   const { sendEvent } = usePreferencesAnalytics();
-  const { logMetadata, preferences } = useLogContext();
+  const { preferences } = useLogContext();
 
   const { prettyPrint, setPrettyPrint } = preferences;
-  const disablePrettyPrint =
-    logMetadata?.renderingType !== LogRenderingTypes.Resmoke;
 
   return (
     <BaseToggle
       data-cy="pretty-print-toggle"
-      disabled={disablePrettyPrint}
       label="Pretty Print Bookmarks"
       onChange={(value) => {
         setPrettyPrint(value);
         sendEvent({ name: "Toggled pretty print", on: value });
       }}
-      tooltip="Toggle pretty printing for bookmarked lines. Only available for resmoke logs."
+      tooltip="Toggle pretty printing for bookmarked lines."
```

Comment:
[nit] I think the tooltip should give information about the feature, not the toggle
```suggestion
      tooltip="Pretty print bookmarked lines."
```